README.md

```diff
@@ -78,6 +78,14 @@ There are many more parameters you can use. Display the usage doc with the follo
 
     GitLink.exe -h
 
+### Native PDBs
+
+Native PDBs (from C++ projects) are supported by using -a option:
+
+    GitLink.exe <nativePdbfile> -a
+
+All source code files from your git depot will be indexed in the PDB.
```

Comment:
>  "all known C++ source files."

Clearly better.

> That would be great. Otherwise for PDBs that belong to very large repos, this could severely bloat the PDB files and significantly slow the build as it indexes tens of thousands of source files for each project that uses gitlink, even though each individual project may only include a small fraction of those.

Yes that's not ideal at all but at least it allows basic support for C++ projects.